data/gurjeetsinghvirdee.json

```diff
@@ -20,8 +32,7 @@
       "icon": "FaTwitter"
     },
     {
-      "name": "Add me on Discord",
-      "url": "Guri#6145",
+      "name": "Add me: Guri#6145",
       "icon": "FaDiscord"
     }
```

Comment:
I'm not sure if it will work without the url

Reply:
for now, removing this
I will work on this later